bin/csv2avro

```diff
@@ -48,11 +48,12 @@ begin
 
   CSV2Avro.new(options).convert
 rescue OptionParser::MissingArgument => ex
-  puts ex.message
-
-  puts option_parser
+  $stderr.puts ex.message
+  $stderr.puts option_parser
+  exit 2
 rescue Exception => e
-  puts 'Uh oh, something went wrong!'
-  puts e.message
-  puts e.backtrace.join("\n")
+  $stderr.puts 'Uh oh, something went wrong!'
+  $stderr.puts e.message
+  $stderr.puts e.backtrace.join("\n")
+  exit 2
```

Comment:
A typical case of `exit 1`, something went wrong is exit code 1 pretty much everywhere:

https://www.youtube.com/watch?v=7QLSRMoKKS0